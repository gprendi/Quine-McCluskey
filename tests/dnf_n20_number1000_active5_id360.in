d 20
00000001010000000000
00001001100000000000
00000000010000100000
00001001110000100000
00001001000000100000
00000000100000000000
00001001000000000000
00001000100000100000
00001000010000100000
00001000010000000000
00000001100000100000
00000001100000000000
00000000000000000000
00000001000000000000
00001000110000100000
00000000000000100000
00001000100000000000
00000001110000000000
00000000110000000000
00001001110000000000
00001001010000100000
00000000110000100000
00001001010000000000
00001000000000000000
00000000010000000000
00000001000000100000
00000001010000100000
00001001100000100000
00000000100000100000
00000001110000100000
00001000110000000000
00001000000000100000
