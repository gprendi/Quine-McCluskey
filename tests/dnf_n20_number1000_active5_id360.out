p dnf 20 0
-1 -2 -3 -4 -6 -7 -11 -12 -13 -14 -16 -17 -18 -19 -20 0
